actix-broker/CHANGES.md

```diff
@@ -1,7 +1,7 @@
 # Changes
 
-## Unreleased - 2021-xx-xx
-
+## Unreleased - 2022-xx-xx
+- Fix at-least-once delivery guarantee for IssueAsync
```

Comment:
```suggestion
- Fix at-least-once delivery guarantee for IssueAsync. [#536]

[#536]: https://github.com/actix/actix/pull/536
```

Reply:
I would usually add this kind of thing myself but you haven't enabled "allow maintainer edits" for this PR.